tests/sentry/eventstore/snuba/test_backend.py

```diff
@@ -94,6 +95,27 @@ def test_get_events(self):
         assert events[1].event_id == "b" * 32
         assert events[2].event_id == "a" * 32
 
+    def test_get_events_with_set_list(self):
+        events = self.eventstore.get_events(
+            filter=Filter(
+                project_ids=[self.project1.id, self.project2.id],
+            ),
+            event_list=[
+                Event(
+                    project_id=self.project1.id,
+                    event_id="a" * 32,
+                ),
+                Event(
+                    project_id=self.project2.id,
+                    event_id="b" * 32,
+                ),
```

Comment:
Seems curious to be giving a list of events to eventstore to get the events. Could there be another structure we use instead? Something like an `EventReference` or `EventIdentifier`?

Reply:
Hmm good point :+1: I'll create `EventIdentifier`, I think get_events can return stubbed events which might get confused with `EventReference`